src/lib/swap-router/utils/fee.utils.ts

```diff
@@ -0,0 +1,34 @@
+import { BigNumber } from 'bignumber.js';
+
+import { DexTypeEnum } from '../enum/dex-type.enum';
+import { RoutePairWithDirection } from '../interface/route-pair-with-direction.interface';
+import { Trade, TradeOperation } from '../interface/trade.interface';
+
+const PAIR_FEE_PERCENT_RECORD: Record<DexTypeEnum, number> = {
+  [DexTypeEnum.QuipuSwap]: 0.3,
+  [DexTypeEnum.Plenty]: 0.35,
+  [DexTypeEnum.LiquidityBaking]: 0.21
+};
+
+export const getPairFeeRatio = (pair: RoutePairWithDirection) => {
+  const feePercent = PAIR_FEE_PERCENT_RECORD[pair.dexType];
+
+  return new BigNumber(100).minus(feePercent).dividedBy(100);
```

Comment:
in other file there is `const HUNDRED`. Reuse same style here (no magic values) or there (no constant approach)